### PR TITLE
Allow validate with functions, allow repeat

### PR DESCRIPTION
**Pull request by baranovxyz**
Both validate with function and repeats could be used in some cases.

### CODE REVIEW

File: src/InputTag.vue [hunk 1]
```diff
@@ -102,6 +107,8 @@ export default {
         return true
       } else if (typeof (this.validate) === 'string' && Object.keys(validators).indexOf(this.validate) > -1) {
         return validators[this.validate].test(tagValue)
+      } else if (typeof (this.validate) === "function" && this.validate.call !== undefined) {
```
Comment (matiastucci) [inline src/InputTag.vue]: why do we need to call it using `.call()`?

**Comment by matiastucci** [comment]
Thanks! can you add tests for this and update the README, please?

**Comment by baranovxyz** [comment]
I'll try.

File: src/InputTag.vue (hunk 1 above)
Comment (matiastucci) [reply src/InputTag.vue]: Got it. I think calling it like `this.validate(tagValue)` it's ok 👍

**Comment by baranovxyz** [comment]
Do you use prettier or something? I messed up formatting a bit. Is there some command to run?

**Comment by matiastucci** [comment]
Nope. Just linter. Did you try `npm run lint:fix`?

**Comment by baranovxyz** [comment]
Closed in favor of #72

**Pull request closed without merging**